src/environments/environment.ts

```diff
@@ -5,12 +5,12 @@
 export const environment = {
   production: false,
   firebase: {
-    apiKey: 'AIzaSyDWKZmGX7RaxDfv8_s_mqA_Ct9iFSP2GiM',
```

Comment:
Is the environment file for staging missing?

Reply:
We already have a environment.staging.ts. I just changed this file to point to the common developer firebase.